dist/combobox/ds4/combobox.css

```diff
@@ -76,6 +76,16 @@ span.combobox {
 .combobox__option[role^="option"]:hover {
   background-color: #eee;
 }
+.combobox__option[role^="option"] svg.icon {
+  align-self: center;
+  fill: currentColor;
+  height: 8px;
+  margin: 0 auto;
+  opacity: 0;
+  stroke: currentColor;
+  stroke-width: 0;
+  width: 8px;
+}
```

Comment:
Shall we place this block of code in `src/less/combobox/base/combobox.less` so that it reflects in `ds4` and `ds6` (Also for other changes you have in this PR where ever possible)?

Reply:
It is in base. Those are the “compiled” files that output on build. One for ds4 and one for ds6

Reply:
My bad! Missed it.

Reply:
I have a separate question, why do we checkin dist folder? We should ideally have a pipeline that should build and distribute the artifacts

Reply:
This is how we always did it. I think it was done like this so anyone could download the repo and get going without any build step. 
However Ian knows more of the history than I do

Reply:
Good question. The main reason for this is so that we can manually diff the CSS files. Given that the LESS source files often don't resemble the final generated CSS, I feel it's important to be able to diff the actual CSS that goes to the browser. If we could all agree to write our LESS more like CSS, then I might be able to remove this part of the process :-) But even then we have things like autoprefixer being added which I think are important to check.